src/main.asm

```diff
@@ -116,8 +116,8 @@ SwitchBlockTiles::
 incbin "gfx/items/switch_block.dmg.2bpp"
 MushroomTiles::
 incbin "gfx/items/mushroom.dmg.2bpp"
-CyclopKeyTiles::
-incbin "gfx/items/cyclop_key.dmg.2bpp"
+SlimeKeyTiles::
+incbin "gfx/items/Slime_key.dmg.2bpp"
```

Comment:
As you can see, the automated build is failing. If you click on "Details" near the "1 failing check: workflow", you'll be redirected to CircleCI, which will show you the build error message:

>     Error opening INCBIN file 'gfx/items/Slime_key.dmg.2bpp': No such file or directory

I guess you renamed the path to the file in the code, but forgot to rename the file itself (so now it's trying to find a file that doesn't exist).

If you rename the file itself, the build should complete successfully. (Also, even better, you can keep the file name all lowercased: `slime_key.dmg.2bpp` instead of `Slime_key.dmg.2bpp)

Reply:
ok , I will try to solve this issue